lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3047,31 +3042,24 @@ prop_distributeSurplusDelta_coversCostIncreaseAndConservesSurplus
 -- Properties for 'distributeSurplus'
 --------------------------------------------------------------------------------
 
-instance Arbitrary FeePolicy where
+instance Arbitrary FeePerByte where
     arbitrary = frequency
-        [ (1, feePolicyMainnet)
-        , (7, feePolicyGeneral)
+        [ (1, pure mainnetFeePerByte)
+        , (7, anyFeePerByte)
         ]
       where
-        feePolicyMainnet :: Gen FeePolicy
-        feePolicyMainnet = pure $ LinearFee $ LinearFunction
-            {intercept = 150_000, slope = 44}
-
-        feePolicyGeneral :: Gen FeePolicy
-        feePolicyGeneral = do
-            intercept <- frequency
-                [ (1, pure 0.0)
-                , (3, getPositive <$> arbitrary)
-                ]
-            slope <- frequency
-                [ (1, pure 0.0)
-                , (3, getPositive <$> arbitrary)
+        anyFeePerByte :: Gen FeePerByte
+        anyFeePerByte = FeePerByte <$>
+            frequency
+                [ (1, pure 0)
+                , (3, fromIntegral @Int @Natural . getPositive <$> arbitrary)
```

Comment:
Perhaps this would also work:

```suggestion
                , (3, (+ 1) <$> arbitrarySizedNatural)
```

Reply:
But since `arbitrarySizedNatural` generates values of `0` some of the time, perhaps it would be enough to write:
```hs
anyFeePerByte = FeePerByte <$> arbitrarySizedNatural
```

The magnitude depends on the QC size parameter, which is probably what we want here.

Reply:
> anyFeePerByte = FeePerByte <$> arbitrarySizedNatural

Yup, should be enough 👍 

23d5b97fc937da2cf66d28b013c3c3a155e3ef02